python/cudf_polars/tests/test_union.py

```diff
@@ -22,19 +21,6 @@ def test_union():
     assert_gpu_result_equal(query)
 
 
-def test_union_schema_mismatch_raises():
```

Comment:
This fails in polars now
```
polars.exceptions.InvalidOperationError: 'union'/'concat' inputs should all have the same schema,got
Schema:
name: a, field: Int64
name: b, field: Int64
 and 
Schema:
name: a, field: Float32
name: b, field: Int64


Resolved plan until failure:

        ---> FAILED HERE RESOLVING THIS_NODE <---
simple π 2/2 ["a", "b"]
   WITH_COLUMNS:
   [null.cast(Int64).alias("b")] 
     SELECT [col("a").strict_cast(Float32)] FROM
      DF ["a", "b"]; PROJECT */2 COLUMNS
```

Reply:
Ah great, so if we update to a more recent polars (as a minimum version) we can just remove that handling. Or we can condition this test on the polars version.